test/TestApps/PushNotificationsTestAppPackage/PushNotificationsTestAppPackage.wapproj

```diff
@@ -42,7 +42,13 @@
     <EntryPointProjectUniqueName>..\PushNotificationsTestApp\PushNotificationsTestApp.vcxproj</EntryPointProjectUniqueName>
     <GenerateAppxPackageOnBuild>true</GenerateAppxPackageOnBuild>
     <GenerateAppInstallerFile>False</GenerateAppInstallerFile>
-    <PackageCertificateKeyFile>$(SolutionDir)temp\MSTest.pfx</PackageCertificateKeyFile>
+    <PackageCertificateKeyFile>$(SolutionDir).user\winappsdk.certificate.test.pfx</PackageCertificateKeyFile>
```

Comment:
Are these project specific?  Couldn't this be done as a centralized target for the whole tree rather than in each project file?  We already have places for that type of thing.